docs/content/en/docs/Usage/_index.md

```diff
@@ -1,86 +1,397 @@
 ---
 title: Usage
 description: Use Azure Quick Review &mdash; to analyze Azure resources and identify whether they comply with Azure's best practices and recommendations.
-weight: 2
+weight: 3
 ---
 
 ## Authentication
 
-**Azure Quick Review (azqr)** supports the following authentication methods:
+### - PowerShell
 
-* Service Principal. You'll need to set the following environment variables:
-  * AZURE_CLIENT_ID
-  * AZURE_CLIENT_SECRET
-  * AZURE_TENANT_ID
-* Azure Managed Identity
-* Azure CLI (Using this type of authentication will make scans run slower)
+> Install Azure PowerShell Modules
 
-## Authorization
+```powershell
+Install-Module -Name 'Az' --Scope 'CurrentUser'
+```
+
+> Create Service Principal
+
+``` powershell
+$spDetails = New-AzADServicePrincipal -DisplayName 'sp-azure-quick-review'
+```
+
+From `$spDetails`, Extract the
+
+- AppId..........: `$spDetails.appId`
+- AppSecret..: `$spDetails.PasswordCredentials.SecretText`
+- TenantId....: `(Get-AzContext).Tenant.Id`
+
+> Authenticate to Azure
+
+``` powershell
+$tenantId = ''
+$appId = ''
+$appSecret = '' | ConvertTo-SecureString -AsPlainText -Force
+$appCreds = New-Object System.Management.Automation.PSCredential($appId , $appSecret)
+
+Connect-AzAccount -Tenant $tenantId -ServicePrincipal -Credential $appCreds
```

Comment:
This is actually slower than using the environment variables:
  * AZURE_CLIENT_ID
  * AZURE_CLIENT_SECRET
  * AZURE_TENANT_ID
  
 So instead of using `Connect-AzAccount` set the env variables.